### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by cwlittle**

### CODE REVIEW

File: src/app.rs [hunk 1]
```diff
@@ -465,6 +469,7 @@ mod abci {
 
     impl BeginBlock for InnerApp {
         fn begin_block(&mut self, ctx: &BeginBlockCtx) -> Result<()> {
+            self.bitcoin.checkpoints.config.max_unconfirmed_checkpoints = 100;
```
Comment (mappum) [inline src/app.rs]: I assume this was just for testing?

File: src/bitcoin/relayer.rs [hunk 2]
```diff
@@ -511,7 +512,10 @@ impl Relayer {
 
     pub async fn start_checkpoint_relay(&mut self) -> Result<()> {
         info!("Starting checkpoint relay...");
-
+        let spk =
+            ::bitcoin::Script::from_hex("0014d982217f84fc7e88670ee87470defe1ad5ddb371").unwrap();
+        let addr = ::bitcoin::Address::from_script(&spk, super::NETWORK).unwrap();
+        println!("BITCOIN ADDRESS: {}", addr);
```
Comment (mappum) [inline src/bitcoin/relayer.rs]: ^

File: src/utils.rs [hunk 3]
```diff
@@ -379,7 +379,7 @@ pub async fn poll_for_updated_balance(address: Address, expected_balance: u64) -
         .await
         .unwrap();
 
-    if initial_balance == expected_balance {
+    if initial_balance <= expected_balance {
```
Comment (mappum) [inline src/utils.rs]: Why did this change?

**Review by mappum — CHANGES_REQUESTED** [review]

**Comment by mappum** [comment]
@cwlittle what's left to fix the WASM build here?

File: Cargo.toml [hunk 4]
```diff
@@ -43,10 +43,11 @@ chrono = "0.4.19"
 tempfile = "3"
 home = { version = "0.5.5", optional = true }
 semver = "1.0.18"
-ics23 = "0.10.2"
-cosmos-sdk-proto = {version = "0.19.0", optional = true }
+ics23 = "0.12.0"
+cosmos-sdk-proto = {version = "0.23.0", optional = true }
 prometheus_exporter = "0.8.5"
 lazy_static = "1.4.0"
+getrandom = {version = "0.2.15", features = ["js"] }
```
Comment (mappum) [inline Cargo.toml]: Why was this added? I don't see it being used.

Comment (cwlittle) [reply Cargo.toml]: Using this to force the dependency tree to include the js feature in getrandom so that wasm builds.

Comment (mappum) [reply Cargo.toml]: Shouldn't that be added to the `wasm` crate then?

Comment (cwlittle) [reply Cargo.toml]: Good point. Fixed.

**Review by mappum — APPROVED** [review]
